proto/net.proto

```diff
@@ -81,14 +81,45 @@ message TrainingParams {
   optional string lc0_params = 6;
 }
 
+message NetworkFormat {
+  // Format to encode the input planes with. Used by position encoder.
+  enum InputFormat {
+    INPUT_UNKNOWN = 0;
+    INPUT_CLASSICAL_112_PLANE = 1;
+  }
+  optional InputFormat input = 1;
+
+  // Output format of the NN. Used by search code to interpret results.
+  enum OutputFormat {
+    OUTPUT_UNKNOWN = 0;
+    OUTPUT_CLASSICAL = 1;
+    // OUTPUT_WDL = 2;  // Uncomment when implemented.
```

Comment:
There are two kinds of output - value head and policy head.  This commented enum value suggests that this enum is actually specifically for the value head axis.  So maybe it should be called ValueOutputFormat and value_output.

(That being said given the network structure it should be obvious (from the number of bytes in the last layer of the value head) whether its a 3 valued output or a 1 valued output so this enum could be avoided entirely unless we plan on having some very strange value head options in future.)

Reply:
As discussed, will keep just one enum for output, but will change GetPVal and GetQVal API to just be GetOutputVal()